client/src/pages/Login/Login.tsx

```diff
@@ -39,18 +38,20 @@ export default function Login(): JSX.Element {
   return (
     <Grid container component="main" className={classes.root}>
       <CssBaseline />
-      <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
+      <Grid item xs={12} sm={12} md={12} elevation={6} component={Paper} square>
         <Box className={classes.authWrapper}>
-          <AuthHeader linkTo="/signup" asideText="Don't have an account?" btnText="Create account" />
-          <Box width="100%" maxWidth={450} p={3} alignSelf="center">
-            <Grid container>
-              <Grid item xs>
-                <Typography className={classes.welcome} component="h1" variant="h5">
-                  Welcome back!
-                </Typography>
+          <NavBar loggedIn={false} user={undefined} />
```

Comment:
Now that this is a flexible component, it would be better to add this component once for the whole app rather than add it individually for each page. We can also receive `loggedIn/user` state from `AuthContext` instead of manually providing it 👍🏼